pallets/phala/src/compute/wrapped_balances.rs

```diff
@@ -499,5 +732,12 @@ pub mod pallet {
 			let vote_info = pallet_democracy::Pallet::<T>::referendum_info(vote_id);
 			matches!(vote_info, Some(ReferendumInfo::Ongoing(_)))
 		}
+
+		fn get_election_lock(who: &T::AccountId) -> BalanceOf<T> {
```

Comment:
It's the combination of all the locks. So I think `get_lock` is a better name.

Reply:
done